kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h

```diff
@@ -125,19 +134,65 @@ class ResidualBasedBlockBuilderAndSolver
         // Validate default parameters
         Parameters default_parameters = Parameters(R"(
         {
-            "name" : "ResidualBasedBlockBuilderAndSolver"
+            "name"                               : "ResidualBasedBlockBuilderAndSolver",

```

Comment:
i think that in PR about factories for strategies a proposal was made to use something other than "name" for this.